ConnectorRhino/ConnectorRhino/ConnectorRhinoShared/UI/MappingBindingsRhino.cs

```diff
@@ -158,6 +161,7 @@ private List<Schema> GetObjectSchemas(RhinoObject obj)
         }
       }
     }
+    // todo: this exception needs to be investigated further
```

Comment:
It feels like we should add a ticket to look into this, then we can add comments referencing the ticket# this I would prefer - CNX-9001